### ISSUE TITLE
Script tag usage of @jbrowse/react-linear-genome-view fails in v1.7.3

### ISSUE THREAD

**Issue by cmdcolin** [GMOD/jbrowse-components#2924]
I wanted to get a "vanilla JS"/script tag include usage of the embedded components on the demos but from what I can tell, this usage is working in v1.6.9 but not working in v1.7.3

WIP vanillajs demos here https://github.com/GMOD/jbrowse-components/tree/vanillajsdemo

The error from v1.7.3 is

```
index-esm.js:1 Uncaught ReferenceError: returntrue is not defined
    at mr (index-esm.js:1:16544)
    at ls (index-esm.js:16:129454)
    at Module.../../node_modules/@mui/x-data-grid/dist/index-esm.js (index-esm.js:16:129852)
    at __webpack_require__ (bootstrap:19:1)
    at Module.../../packages/core/ReExports/modules.ts (material-ui-colors.js:18:1)
    at __webpack_require__ (bootstrap:19:1)
    at Module.../../packages/core/ReExports/index.ts (PluginManager.ts:584:3)
    at __webpack_require__ (bootstrap:19:1)
    at Module.../../packages/core/PluginManager.ts (PluginLoader.ts:246:3)
    at __webpack_require__ (bootstrap:19:1)
mr @ index-esm.js:1
ls @ index-esm.js:16
../../node_modules/@mui/x-data-grid/dist/index-esm.js @ index-esm.js:16
__webpack_require__ @ bootstrap:19
../../packages/core/ReExports/modules.ts @ material-ui-colors.js:18
__webpack_require__ @ bootstrap:19
../../packages/core/ReExports/index.ts @ PluginManager.ts:584
__webpack_require__ @ bootstrap:19
../../packages/core/PluginManager.ts @ PluginLoader.ts:246
__webpack_require__ @ bootstrap:19
./src/createModel/createModel.ts @ createConfigModel.ts:44
__webpack_require__ @ bootstrap:19
./src/createModel/index.ts @ createSessionModel.ts:364
__webpack_require__ @ bootstrap:19
(anonymous) @ node module decorator:5
(anonymous) @ index.ts:4
(anonymous) @ index.ts:4
webpackUniversalModuleDefinition @ universalModuleDefinition:9
(anonymous) @ universalModuleDefinition:10
genomeView.js:6 Uncaught ReferenceError: JBrowseReactLinearGenomeView is not defined
    at genomeView.js:6:3
```

Inside the UMD bundle there is, indeed, in v1.7.3 a weird `returntrue` variable that would likely trigger this

![Screenshot from 2022-04-18 13-12-35](https://user-images.githubusercontent.com/6511937/163862585-cc2ddc08-f674-4c55-a87c-582186485efc.png)

**Comment by rbuels** [comment]
whoa that's a crazy one

**Comment by cmdcolin** [comment]
indeed. looks like there is another mention of it here https://github.com/mui/mui-x/issues/4490

**Comment by cmdcolin** [comment]
will be fixed in next by #2925 which required pinning webpack 5.27.0
